### DISCUSSION TITLE
I did not understand the benefit of Pipelines

### DISCUSSION THREAD

**Discussion by alanqoudif** [open-webui/pipelines#147] [Q&A]
I did not understand the benefit of Pipelines even though I mainly use open webui and ollama.

**Comment by hasani114** [comment]
Pipelines are extremely powerful and they can be used to create custom workflows that are independent from each other and work as separate "Agents" of sorts.

One use case that I'm working on is creating different pipelines for tasks / specializations, for example, one pipeline speciaizes in communication and it will have an agent that can access tools to send emails, google chat messages, etc., Another pipeline will have access to our product bug tracking software so it can look up open issues, another pipeline will have access to our CRM, and so on.

When you have multiple pipelines you can invoke them using the @ and selecting the pipeline you want to "talk to". A most automated and advance use case that I test is to use a filter that decides which pipeline should be used based on the question.

Once this is done, my workflow will look like this.

ask AI to look up open issues / newly reported issues. >>> mantis/jira pipeline
ask AI to see if there is a workaround or way to resolve >>> documentation pipeline
ask AI to email/message product manager for that specififc product or the dev/qa >>> communication pipeline

etc etc 

the benefits are endless.
